Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -77,35 +77,12 @@ public async Task<bool> ExecuteAsync(int retryCount, string dumpRootPath)
 
             if (_hasSetupOperation)
             {
-                var snapshotSubdirectory = "SQLite";
-                if (_configuration.UseSqlServer)
-                {
-                    snapshotSubdirectory = _configuration.UseAzureBlobStorage
-                        ? "SqlServer-AzureBlob"
-                        : "SqlServer";
-                }
-                else if (_configuration.UseAzureBlobStorage)
-                {
-                    snapshotSubdirectory = "SQLite-AzureBlob";
-                }
-
-                snapshotSubdirectory += "-" + setupConfiguration.SetupOperation!.GetHashCode().ToTechnicalString();
-
-                _snapshotDirectoryPath = Path.Combine(setupConfiguration.SetupSnapshotDirectoryPath, snapshotSubdirectory);
-
-                _configuration.OrchardCoreConfiguration.SnapshotDirectoryPath = _snapshotDirectoryPath;
-
-                _currentSetupSnapshotManager = UITestExecutionSessionsMeta.SetupSnapshotManagers.GetOrAdd(
-                    _snapshotDirectoryPath,
-                    path => new SynchronizingWebApplicationSnapshotManager(path));
-
```

Comment:
I just moved this due to the cognitive complexity of the method becoming too large (analyzer).